Makefile

```diff
@@ -50,3 +50,11 @@ clear-cache:
 	docker rm -vf $$(docker ps -a -q) 2>/dev/null ; \
 	docker rmi -f $$(docker images -a -q) 2>/dev/null ; \
 	docker volume rm $$(docker volume ls -q) 2>/dev/null ; true
+
+experiment=*
+
+stop-trials:
```

Comment:
Please put this in stop_experiment.py.
I don't really think this should have a make target either frankly, but that's a separate point.

Reply:
My understanding was that this "stop-trials" is not the official way to stop an experiment, but a quick temporary fix only for developers to clean up stuck/stale/leftover/zombie trials that often happen currently due to some issues. Is that correct?

If yes, we might consider also calling this accordingly, e.g. as `kill-trials`, and leave a comment explaining the purpose and when to use it.

Reply:
Sure. Added to it and waiting for a last local test.
I think the existing of `make clear-cache` letting us believe that it is fine to have other cleaning options. If `stop-trials` is not needed or not proper, it would be better remove `clear-cache` as well.

Reply:
> My understanding was that this "stop-trials" is not the official way to stop an experiment, but a quick temporary fix only for developers to clean up stuck/stale/leftover/zombie trials that often happen currently due to some issues. Is that correct?

This is for one purpose. Another is for end-to-end testing, where it will cause trials not finished it the experiment has problems.
Let me add some comments for this make target.

Reply:
> kill-trials

Shall we avoid words like `kill`? I think `stop` or `terminate` should be fine and also warmer and sounds peaceful. :)

Reply:
I'd be happy to remove `clear-cache` I think it was a mistake to add it frankly.

Reply:
As discussed offline, let's
1) remove `stop-trials` from Makefile
2) move the `stop-experiment` logic to `stop_experiment.py` (feel free to keep the `stop-experiment` make target, if you think it's useful, that just calls the `experiments/stop_experiment.py` script)
3) also remove the `clear-cache` from Makefile.